### PR TITLE
Release

### PR DESCRIPTION
**Pull request by b-gehrke**

### CODE REVIEW

File: Upper Level BCIO/bcio.owl
Comment (jannahastings) [inline Upper Level BCIO/bcio.owl:1]: The versionIRI is still August 30 (the last release). This is speciifed in the final merge command (the annotate part), manually but could automatically use the last date.

Comment (jannahastings) [reply Upper Level BCIO/bcio.owl:1]: It looks as though IAO_0000078 has not been imported, please check (the annotation property to which the curation status is being attached)

Comment (jannahastings) [reply Upper Level BCIO/bcio.owl:1]: Otherwise looks great! 👍🏻

**Review by jannahastings — APPROVED** [review]

File: Upper Level BCIO/bcio.owl
Comment (b-gehrke) [reply Upper Level BCIO/bcio.owl:1]: > It looks as though IAO_0000078 has not been imported, please check (the annotation property to which the curation status is being attached)

It is. Line 62 in bcio.owl. But I noticed it is used as Annotation Property while in IAO it is defined as a class. Shouldn't we use [has curation status (IAO:0000114)](http://purl.obolibrary.org/obo/IAO_0000114)

Comment (jannahastings) [reply Upper Level BCIO/bcio.owl:1]: Ah, that seems to be a bug then! Yes we should use has curation status (and import that)

Comment (jannahastings) [reply Upper Level BCIO/bcio.owl:1]: Looks brilliant. Good to go!

**Pull request merged**